build.gradle.kts

```diff
@@ -30,7 +28,7 @@ configurations.all {
 
 dependencies {
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion")
-    implementation("org.testcontainers:testcontainers:1.10.5")
+    api("org.testcontainers:testcontainers:1.10.5")
```

Comment:
I'd prefer the PR which kills the dependency than exposing it in API :)

Reply:
I want to expose some Docker API. Otherwise we'd have to cover a wide array of APIs and use cases without providing any clear value.
Whether it's `docker-java` or `testcontainers` is a secondary choice.